frontend/packages/operator-lifecycle-manager/src/components/subscription.tsx

```diff
@@ -320,22 +331,50 @@ export const SubscriptionsPage: React.FC<SubscriptionsPageProps> = (props) => {
   );
 };
 
-export const SubscriptionDetails: React.FC<SubscriptionDetailsProps> = ({
-  catalogSources = [],
-  clusterServiceVersions = [],
-  installPlans = [],
-  obj,
-  packageManifests = [],
-  subscriptions = [],
-}) => {
+const CatalogSourceHealthAlert = ({ health, source, sourceNamespace }) => {
+  const { t } = useTranslation();
+  if (!health) {
+    return (
+      <Alert
+        isInline
+        className="co-alert"
+        variant="warning"
+        title={t('olm~CatalogSource health unknown')}
+      >
+        {t(
+          'olm~This operator cannot be updated. The health of CatalogSource "{{source}}" is unkown. It may have been disabled or removed from the cluster.',
```

Comment:
```suggestion
          'olm~This operator cannot be updated. The health of CatalogSource "{{source}}" is unknown. It may have been disabled, or removed from the cluster.',
```